packages/account-v2/src/containers/ManualForm/ManualForm.tsx

```diff
@@ -28,16 +28,19 @@ export const ManualForm = ({
     const validationSchema = getManualFormValidationSchema(selectedDocument, isExpiryDateRequired);
 
     const initialValues = useMemo(() => {
-        const defaultValues = setInitialValues(Object.keys(validationSchema.fields));
+        const defaultValues = validationSchema.getDefault();
         const formValues = { ...defaultValues, ...formData };
-        delete formValues[MANUAL_DOCUMENT_SELFIE];
+        if (MANUAL_DOCUMENT_SELFIE in formValues) {
+            delete formValues[MANUAL_DOCUMENT_SELFIE];
+        }
```

Comment:
can you explain why this is needed?

Reply:
Will add a comment in the code. The reason for removing the selfie value is because the Container will contain, 
- Document number
- Expiry date
- Documents uploaded
- Selfie image

But this section of the form should deal with only Doc details except selfie image. Hence the value is getting deleted